beacon_node/beacon_chain/src/execution_payload.rs

```diff
@@ -191,10 +192,13 @@ pub async fn validate_merge_block<'a, T: BeaconChainTypes>(
             if is_optimistic_candidate_block(chain, block.slot(), block.parent_root()).await? {
                 debug!(
                     chain.log,
-                    "Optimistically accepting terminal block";
+                    "Optimistically Importing Merge Transition Block";
```

Comment:
Not a big deal, but this type of capitalisation is not consistent with the rest of the project; it's generally `Something along the lines of this` for the primary log lines and then `something like this` for the `"msg"` field.

Reply:
I've adapted the comments to conform to the rest of the codebase :)